### PR TITLE
Code review feedback from pipeline_flow changes

### PR DESCRIPTION
**Pull request by BertKleewein**

### CODE REVIEW

File: azure-iot-device/tests/common/pipeline/test_pipeline_flow.py [hunk 1]
```diff
@@ -0,0 +1,216 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
```
Comment (BertKleewein) [inline azure-iot-device/tests/common/pipeline/test_pipeline_flow.py]: >-------------------------------------------------------------------------- [](start = 2, length = 74)

CodeFlow doesn't recognize this as a rename, but github does.  If you look at the second commit for this PR on the github site, you can see the changes to this file.

**Comment by olivakar** [comment]
>         Continue an operation using a new operation.  This means that the new operation

may be say a `new worker operation` 

---
Refers to: azure-iot-device/azure/iot/device/common/pipeline/pipeline_flow.py:22 in 5b897f5. [](commit_id = 5b897f504af94d792445fab7ec1eeda9153f242a, deletion_comment = False)

File: azure-iot-device/tests/common/pipeline/test_pipeline_flow.py [hunk 2]
```diff
@@ -0,0 +1,216 @@
+# -------------------------------------------------------------------------
```
Comment (cartertinney) [inline azure-iot-device/tests/common/pipeline/test_pipeline_flow.py]: Seems like there's still no tests for ._send_event_up()

File: azure-iot-device/tests/common/pipeline/test_pipeline_flow.py [hunk 3]
```diff
@@ -0,0 +1,216 @@
...
+def stage(mocker, arbitrary_exception, arbitrary_base_exception):
+    return make_mock_stage(
+        mocker=mocker,
+        stage_to_make=MockPipelineStage,
+        exc_to_raise=arbitrary_exception,
+        base_exc_to_raise=arbitrary_base_exception,
+    )
+
+
+@pytest.mark.describe("PipelineFlow._send_worker_op_down()")
```
Comment (cartertinney) [inline azure-iot-device/tests/common/pipeline/test_pipeline_flow.py]: >PipelineFlow._send_worker_op_down() [](start = 23, length = 35)

The unit syntax we defined would correctly be

`PipelineFlow - ._send_worker_op_down()`

File: azure-iot-device/tests/common/pipeline/helpers.py [hunk 4]
```diff
@@ -134,7 +134,7 @@ def assert_callback_failed(op, callback=None, error=None):
     callback_error_arg = callback.call_args[1]["error"]
     if error:
         if isinstance(error, type):
-            assert isinstance(callback_error_arg, error)
+            assert callback_error_arg.__class__ == error
```
Comment (cartertinney) [inline azure-iot-device/tests/common/pipeline/helpers.py]: >callback_error_arg.__class__ == error [](start = 19, length = 37)

GOOD CHANGE

**Pull request merged**